vignettes/why-it-works.Rmd

```diff
@@ -0,0 +1,191 @@
+
+# Statistical model used in `primarycensoreddist`
+
+As described in [Introduction](#introduction), `primarycensoreddist` focuses on a subset of methods from time to event analysis that address data missingness problems commonly found in epidemiological datasets. We present the statistical problem as a double interval censoring problem, where both the primary event time and the secondary event times are interval censored. We can recover single interval censoring problems by reducing one of the intervals to a point.
+
+## Survival function of secondary event time after primary event window
+
+A key assumption we make is that the censoring window for events is known and independent of the event time. This is known as non-informative censoring. It is useful to consider the time from the end (right) point of the primary censoring interval to the secondary time as a random variable,
+
+$$
+S_+ = T - C_P.
+$$
+
+Where $T$ is the delay distribution of interest and $C_P$ is interval between the end (right) point of the primary censoring window and the primary event time; note that by definition $C_P$ is not observed.
+
+With non-informative censoring, it is possible to derive the upper distribution function of $S_+$, or _survival function_ of $S_+$, from the distribution of $T$ and the distribution of $C_P$.
+
+$$
+Q_{S_+}(t) = \mathbb{E}_{C_P} \Big[Q_T(t + C_P)\Big] = \int_0^{W_P} Q_T(t + p) c(p) dp.
+$$
+
+Using integration by parts gives:
+$$
+Q_{S_+}(t) = Q_T(t + W_P) + \int_0^{W_P} f_T(t+p) C(p) dp.
+$$
+
+Where $c$ is the density function of delay from the primary event to the end (right) point of the primary censoring window with distribution function $C$, $Q_T$ is the survival function of the actual delay distribution of interest and $W_P$ is the length of the primary censoring window.
+
+## Probability mass of secondary event time within a secondary censoring window
+
+The probability of the secondary event time falling within a secondary censoring window of length $W_S$ that begins at time $n$ _after_ the primary censoring window is then given by:
+
+$$
+Pr(S_+ \in [n, n + W_S)) = Q_{S_+}(n) - Q_{S_+}(n + W_S).
+$$
+
+Which can in general be calculated by numerical quadrature.
+
+Note that the secondary event time can also occur within the primary censoring window. This happens with probability,
+$$
+Q_{S_+}(-W_P) - Q_{S_+}(0) = 1 - Q_T(W_P) - \int_0^{W_P} f_T(p) C(p) dp = Pr(T< C).
+$$
+
+Using a difference operator defined as:
+
+$$
+\Delta_{W}f(t) = f(t + W) - f(t).
+$$
+
+The common case where every primary and secondary event window is a single time unit, $W_P = W_S = 1$, simplifies the above expressions to:
+
+$$
+\begin{aligned}
+f_n = Pr(S_+ \in [n, n + 1)) &= Q_{S_+}(n) - Q_{S_+}(n + 1), \\
+& = - \Delta_1 Q_{S_+}(n),\qquad n = -1, 0,\dots.
+\end{aligned}
+$$
+Which is a discrete probability mass function of the secondary event time relative to the primary event time in units of the event window.
+
```

Comment:
We put a lot of effort in to tell people about truncation and now its a secret. I thnk this might work best as a standalone section which shows how truncation fits into the above ideas (i.e. its like any other truncation approach and all thee special clever SamB sauce can be ignored at this stage).